core/templates/dev/head/pages/admin/roles_tab/role_graph_directive.html

```diff
@@ -0,0 +1,70 @@
+<!-- NOTE TO DEVELOPERS: The svg defs needed for this directive are
+included at the bottom of exploration_editor.html. In order for the
```

Comment:
If it looks fine, it's probably OK to do so. In the exploration editor the same "graph component" is used for graphs in the editor, stats and history tabs, so perhaps that's why this notice was there.